Jenkinsfile

```diff
@@ -132,27 +110,9 @@ VERSION      : ${env.VERSION}
           version = "${packageJson.version}-${featureBranch}.${env.BUILD_NUMBER}".toString()
         }
 
-        def npmFunctions = null
-        def gitFunctions = null
```

Comment:
I moved these above to match our other Jenkinsfiles to get these via `new com.genesys.jenkins.Npm()` or similar.